package.json

```diff
@@ -11,6 +11,7 @@
   "scripts": {
     "start": "babel-node ./src/index.js",
     "dev": "nodemon --exec babel-node ./src/index",
+    "prod": "nodemon --exec babel-node ./src/index",
```

Comment:
This is not necessary, you can use the `start script` because we do not need nodemon in production.

Reply:
Yes, we have said that we will change it after. but I will change it